addons/base_setup/views/res_config_settings_views.xml

```diff
@@ -354,6 +354,21 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-12 col-lg-6 o_setting_box" id="recaptcha">
+                                <div class="o_setting_left_pane">
+                                    <field name="module_google_recaptcha"/>
+                                </div>
+                                <div class="o_setting_right_pane" id="website_recaptcha_settings">
+                                    <label for="module_google_recaptcha"/>
+                                    <a href="https://www.google.com/recaptcha/intro/v3.html" title="Documentation" class="o_doc_link" target="_blank"></a>
```

Comment:
The use of this link is reserved for the internal documentation (odoo.com) only for now.
You can move the link below the "=> Generate Key"

Reply:
:crab:

Reply:
Removed it